src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppPlugin.scala

```diff
@@ -120,8 +120,13 @@ object SbtReactiveAppPlugin extends AutoPlugin {
         Seq(file)
       }.taskValue,
 
-      javaOptions in SbtNativePackager.Universal ++= Vector(
-        "-Dconfig.resource=app.conf"),
+      javaOptions in SbtNativePackager.Universal ++=
+        Vector(
+          "-Dconfig.resource=app.conf") ++
+          (
+            if (memory.value.isDefined)
```

Comment:
Perhaps consider introducing another flag for the user to opt out and disable these options despite having `memory` set?

Reply:
Thanks - raised as https://github.com/lightbend/sbt-reactive-app/issues/31